sql/parser/builtins.go

```diff
@@ -951,6 +969,29 @@ func round(x float64, n int64) (Datum, error) {
 	return DFloat(y), err
 }
 
+// Pick the greatest (or least value) from a tuple.
+func pickFromTuple(greatest bool, args DTuple) (Datum, error) {
+	g := args[0]
+	// Pick a greater (or smaller) value.
+	for _, d := range args[1:] {
+		var eval Datum
+		var err error
+		if greatest {
+			eval, err = evalComparison(LT, g, d)
+		} else {
+			eval, err = evalComparison(LT, d, g)
+		}
+		if err != nil {
+			return DNull, err
+		}
+		if eval == DBool(true) ||
+			(eval == DNull && g == DNull) {
```

Comment:
Technically you don't have to call `evalComparison` if `g == DNull`, but I suppose that doesn't really matter.